packages/rs-dpp/src/tests/fixtures/get_dashpay_contract_fixture.rs

```diff
@@ -0,0 +1,33 @@
+use std::sync::Arc;
+
+use lazy_static::lazy_static;
+use serde_json::Value;
+
+use crate::{
+    data_contract::{
+        validation::data_contract_validator::DataContractValidator, DataContract,
+        DataContractFactory,
+    },
+    prelude::Identifier,
+    tests::utils::generate_random_identifier_struct,
+    version::{ProtocolVersionValidator, COMPATIBILITY_MAP, LATEST_VERSION},
+};
+
+lazy_static! {
```

Comment:
You should use system contract loader instead (lib/system-contracts)

Reply:
I didn't know, This is good, thanks!, Fixed